src/ui/components/Header/Header.tsx

```diff
@@ -120,7 +119,8 @@ function HeaderTitle({
 
   return (
     <span
-      className={cx(className, "input focus:border-blue-500 focus:ring-primaryAccent", {
+      style={{ outline: "none", background: "inherit" }}
+      className={cx(className, "input m-5 focus:bg-blue-500", {
```

Comment:
I think you might be able to continue using tailwind
https://tailwindcss.com/docs/outline-style#removing-outlines

not sure about the background though

Reply:
It's crazy, I worked on this for ages and just could not get tailwind working. Had to go with style to make it finally take.